lib/rust/profiler/data/src/lib.rs

```diff
@@ -314,8 +325,10 @@ pub struct Metadata<M> {
 /// A timestamp that can be used for distinguishing event order.
 #[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Default)]
 pub struct Mark {
-    seq:  Seq,
-    time: profiler::internal::Timestamp,
+    /// Sequence number of the mark. Used to resolve timestamp collisions.
+    seq:      Seq,
+    /// Time of the mark.
+    pub time: profiler::internal::Timestamp,
```

Comment:
`profiler::internal::Timestamp` is a low-level detail data consumers shouldn't need to deal with; we could provide `Mark::into_ms()`.

Reply:
Makes sense. Changed to expose/use `Mark::into_ms().`